install/sensors.csv

```diff
@@ -855,6 +855,7 @@ Nanometrics,Trillium Compact 120,617,HD71,10,0,,0,0,0,0,0,0,2017-12-14T00:39:45Z
 Nanometrics,Trillium Compact 120,619,HD50,10,0,,0,0,0,0,0,0,2015-11-17T22:46:00Z,2016-07-18T01:54:00Z
 Nanometrics,Trillium Compact 120,619,HD64,10,0,,0,0,0,0,0,0,2017-04-18T00:00:02Z,2019-06-27T01:02:00Z
 Nanometrics,Trillium Compact 120PH-2,10094,WTVZ,10,0,,0,0,0,0,0,0,2024-06-21T01:43:00Z,9999-01-01T00:00:00Z
+Nanometrics,Trillium Compact 120PH-2,10213,PKTZ,10,0,,0,10,0,0,0,0,2024-07-16T22:06:00Z,9999-01-01T00:00:00Z
```

Comment:
do we need to update the azimuth?

Reply:
Yes, I still need to update the orientation here once Conrad has had a chance to look at the surface sensor data

Reply:
Thanks @Thomas-Benson , I'll approve once the azimuth is updated.

Reply:
@Thomas-Benson @mnaguit I've posted orientation results in the main ticket. Thanks.